govstack_api/building_blocks/bb_digital_registries/registries/insuree_registry.py

```diff
@@ -126,27 +38,28 @@ def get_record_field(self, mapped_data, field=None, extension=None):
     def retrieve_filtered_records(self, mapped_data):
         return self.get_record(mapped_data, only_first=False)
 
-    def update_registry_record(self, mapped_data_query, mapped_data_write):
+    def manage_registry_record(self, mutation_name, mapped_data_query, mapped_data_write=None):
         insuree_uuid = self.extract_uuid(mapped_data_query)
         if insuree_uuid:
-            mapped_data_write["uuid"] = insuree_uuid
-            arguments_with_values = self.create_arguments_with_values(mapped_data_write)
-            query = get_update_registry_query(arguments_with_values)
+            data_to_write = mapped_data_write if mapped_data_write else mapped_data_query
+            data_to_write["uuid"] = insuree_uuid
+            default_data_values = self.get_required_data_for_mutation(data_to_write)
+            arguments_with_values = self.create_arguments_with_values(data_to_write)
+            query = self.get_mutation(
+                mutation_name=mutation_name,
+                arguments_with_values=arguments_with_values,
+                default_values=default_data_values
+            )
             self.client.execute_query(query)
             return 200
         else:
             return 404
 
+    def update_registry_record(self, mapped_data_query, mapped_data_write=None):
+        return self.manage_registry_record(self.mutations['update'], mapped_data_query, mapped_data_write)
+
     def create_registry_record(self, mapped_data):
-        insuree_uuid = self.extract_uuid(mapped_data)
-        if insuree_uuid:
-            mapped_data["uuid"] = insuree_uuid
-            arguments_with_values = self.create_arguments_with_values(mapped_data)
-            query = get_create_registry_query(arguments_with_values)
-            self.client.execute_query(query)
-            return 200
-        else:
-            return 404
+        return self.manage_registry_record(self.mutations['update'], mapped_data)
```

Comment:
I see that in several places (here or extract_insuree_records) you have single line functions that are calling another method. While it's good practice put certain parts of code in separate functions it's also good not to add too much of them as it has negative aspect on readibility and in the long term may make it harder for other devs to keep track of where particular actions take place. Consider reducing the number of these single-line functions and integrating their functionality directly where they're called, where it makes sense and doesn't compromise modularity.